app/services/vot/acr_component_values.rb

```diff
@@ -1,105 +1,302 @@
 # frozen_string_literal: true
 
+# rubocop:disable Layout/LineLength
 module Vot
   module AcrComponentValues
+    # A subtype of ComponentValue that will sort by priority (descending) when used in
+    # an Enumerable-like object.
+    class AcrComponentValue < Vot::ComponentValue
+      include Comparable
+
+      attr_reader :name, :description, :implied_component_values, :requirements
+      def initialize(name:, description:, implied_component_values: [], requirements: [])
+        @name = name
+        @description = description
+        @implied_component_values = implied_component_values.freeze
+        @requirements = requirements.inquiry.freeze
+      end
+
+      def <=>(other)
+        Vot::AcrComponentValues::ACRS_BY_PRIORITY_LIST.index(name) <=> Vot::AcrComponentValues::ACRS_BY_PRIORITY_LIST.index(other.name)
+      end
+    end
```

Comment:
I want to eventually add a `strength` field here, but for simply satisfying the requirement to sort the ACR values by a defined priority (whether a string or AcrComponentValue), this works for now.

I'm deferring adding the `strength` field in part because answering the question of how to sort components of equivalent strength requires more thought.